### ISSUE TITLE
train CornetNet and performance.

### ISSUE THREAD

**Issue by liminghuiv** [princeton-vl/CornerNet-Lite#57]
![image](https://user-images.githubusercontent.com/28763807/57327259-50521100-70c3-11e9-83a0-55589c748b37.png)

The testing result is just completely wrong.

Did you test the cornerNet?

**Comment by heilaw** [comment]
Regarding the performance on test set, since COCO does not provide ground truth annotations for the test set, the evaluation script won't work. You need to submit the results to COCO server. The result JSON file can be found in `<CornerNet-Lite dir>/results/CornerNet/500000/testing`.

Regarding the performance on validation set, it looks like you were training CornerNet with a much smaller batch size, which would hurt the network performance. You can either train CornerNet for longer to see if the performance improves or try CornerNet-Saccade which you can train with only 4 1080Ti GPUs.
